trade-finance-manager-ui/server/test-mocks/show-amendments-button-deal-builder.js

```diff
@@ -0,0 +1,35 @@
+import { DEAL_STATUS } from '@ukef/dtfs2-common';
+import { DEAL } from '../constants';
+
+export class AmendmentButtonDealBuilder {
```

Comment:
Am using a builder as they are easier to read

Reply:
Looks lovely - does this need to be ammendments specific? It looks like a pretty generic deal builder to me